frontend/src/component/feature/FeatureStrategy/FeatureStrategyForm/FeatureStrategyForm.tsx

```diff
@@ -389,6 +390,29 @@ export const FeatureStrategyForm = ({
                         />
                     }
                 />
+
+                <ConditionallyRender
+                    condition={
+                        !BuiltInStrategies.includes(strategy.name || 'default')
```

Comment:
In theory the strategy name should never be undefined but that's what our types say. This is probably a very, very legacy hangover from iteration 1 of Unleash when strategies didn't exist. If that's the case then you effectively have a default strategy